### DISCUSSION TITLE
Add the daily contributions widget!

### DISCUSSION THREAD

**Discussion by HaseebKahn365** [community/community#62739] [Other Feature Feedback, Questions, & Ideas]
Please add the heatmap features for profile overview. It was the only reason I downloaded this app

**It would be great to see daily contributions on the Homescreen.**

I am a huge fan of infographics. I was expecting a few graphs and other similar widgets in this app but I couldn't find any. Otherwise the entire app performs well and is built pretty good.

**Comment by tuves** [comment]
Welcome to the GitHub Community, @HaseebKahn365, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by aaarajuan** [comment]
if you want to showcase your daily contribution you can use the bellow method:
1. create a repository on the same username and when creating the repository click on create `README.md` file.
2. use this `[![Al-Arashad Ahmed Rajuan's GitHub activity graph](https://github-readme-activity-graph.vercel.app/graph?username=aaarajuan&theme=github-compact&title_color=51DD5DFF&hide_border=true)` code to add your daily activity stats or commits stats on a graph [ you can change color or custom title by replacing yours ]

> Note: if you want to watch the preview you can go [here](https://camo.githubusercontent.com/43a5eb4068130057091504ed046ffcf16001923be697817e323e7a6af503fd88/68747470733a2f2f6769746875622d726561646d652d61637469766974792d67726170682e76657263656c2e6170702f67726170683f757365726e616d653d61616172616a75616e267468656d653d6769746875622d636f6d70616374267469746c655f636f6c6f723d353144443544464626686964655f626f726465723d74727565). and how it will look like on profile you can check by visiting my profile [@aaarajuan](https://github.com/aaarajuan)

**Comment by HaseebKahn365** [comment]
Hi Alarashad! I think you misunderstood me I was trying to say that there should be a daily contribution widget on the landing page of the Github
app. I know that it's there one on the website but I wanted on the app as well

**Reply by aaarajuan** [reply]
ah, sorry about that. I think it can only be possible if GitHub makes itself on the app.
